js/oop/Factorial.js

```diff
@@ -12,6 +12,23 @@ export class Factorial {
   }
 
   // write your code here
+  loop() {
+    let result = 1;
+
+    for (let index = this.n; index > 0; index--) {
+      result = result * index;
+    }
+
+    return result;
+  }
+
+  recursive() {
+    if (this.n < 2) {
+      return 1;
+    }
+
+    return this.n * new Factorial(this.n - 1).recursive();
+  }
```

Comment:
Please consider a better name for these two functions (loop and recursive) e.g. countUsingLoop, etc. You always have to make sure that your method name is describing what you actually do on that method. Also, pay attention to this review for the rest of the file.